pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1985,12 +1970,14 @@ func GracePeriod(vmi *v1.VirtualMachineInstance) int64 {
 	return gracePeriodSeconds
 }
 
-func needsMorePCIEControlers(vmi *v1.VirtualMachineInstance) bool {
-	const pciSlotHungryMachineType = "q35"
-	if vmi.Spec.Domain.Machine != nil {
-		if vmi.Spec.Domain.Machine.Type != "" {
-			return strings.Contains(vmi.Spec.Domain.Machine.Type, pciSlotHungryMachineType)
-		}
+func DomainInterfaceSpec(ctx *ConverterContext, attachmentName string, tapDeviceName string) api.Interface {
```

Comment:
Please reuse `converter/network.createDomainInterfaces` instead of introducing a new method.

Any, this method should be added to `converter/network` and not here.

Reply:
We have a `network.go` file there, I think it should be placed there.
And I think you need to name it with a more specific focus, as this is not a generic domain interface.